qa/restart-upgrade/build.gradle

```diff
@@ -144,12 +144,15 @@ task testAgainstNewCluster(type: StandaloneRestIntegTestTask) {
     dependsOn "testAgainstOldCluster"
     dependsOn rootProject.tasks.assemble
     useCluster testClusters."${baseName}"
-    if(ext.bwcBundleTest){
-        doFirst {
+
+    doFirst {
+        // This is added to prevent the cluster from getting stuck in yellow state
+        println "Waiting for cluster to stabilize after previous test, including data synchronization on replica shards"
+        Thread.sleep(10000) // 10 seconds delay
```

Comment:
hardcoded value is good first step, we can try to change it to something more sophisticated later

Reply:
Good comment. Another option is to wait for the cluster to be green before upgrading the node to the next version in this gradle file, or we could wait for the cluster to be green after every test before completing it, since node upgrades only occur once the previous test has finished.

Reply:
@heemin32 This would be a better option to wait for cluster to become green as a next phase after we added the hardcoded delay as the first step

Reply:
Yes. Let's see whether yellow cluster issues occur again in the future. For now, this approach should work well since we only need a 1-2 second delay at most, and there's minimal downside to using a fixed delay.